stdpopsim/species.py

```diff
@@ -114,6 +117,14 @@ class Species(object):
     demographic_models = attr.ib(factory=list, kw_only=True)
     genetic_maps = attr.ib(factory=list, kw_only=True)
 
+    @property
+    def ensembl_id(self):
+        """
+        Returns the ID of this species for the Ensembl REST API.
+        This is the species name, underscore delimited and in lowercase.
+        """
+        return self.name.lower().replace(" ", "_")
+
```

Comment:
Is this really needed? The REST API seems to accept space-delimited names, mixed case, taxon id, etc.

Reply:
I dunno, I spent a long time fiddling with it, trying all sorts of things and this was the only thing I could get to work reliably. They do seem to use this form in the docs a lot, so maybe that's their basic internal representation and so less likely to break.